caps/types.go

```diff
@@ -60,6 +63,24 @@ func (t *BoolFileType) Sanitize(c *Capability) error {
 	return nil
 }
 
+// SecuritySnippet for bool-file capability type.
+// Consumers gain permission to read, write and lock the designated file.
+func (t *BoolFileType) SecuritySnippet(c *Capability, securitySystem string) (string, error) {
+	switch securitySystem {
+	case SecurityApparmor:
+		// TODO: switch to the absolute path later
+		path := c.Attrs["path"]
+		// Allow read,write and lock on the file designated by the path.
+		return fmt.Sprintf("%s rwl,\n", path), nil
+	case SecuritySeccomp:
+		return "", nil
```

Comment:
If Seccomp is active, will this capability work if nothing is configured?  If it won't, then we shouldn't return an empty snippet, as that indicates nothing is required for it to work.